docs/manual/developer/06_contributing_with_content.md

```diff
@@ -1887,6 +1887,9 @@ The selected value can be changed in the profile (consult the actual variable fo
     -   **embedded_data** - if set to `"true"` and used combined with `xccdf_variable`, the data retrieved by `yamlpath`
         is considered as a blob and the field `value` has to contain a capture regex.
 
+    -   **all_exists_for** - if set to a YAML Path that points to a set of scalar values then the amount of items collected
+        by the test would be compared to the amount of these "control" items. The test would fail if they are not equal.
+
```

Comment:
I think its relation ship to `yamlpath` is not very clear.

Suggestion:
```suggestion
    -   **all_exists_for** - Use this parameter to ensure that the yaml sequence or array referenced by `yamlpath` is not missing the `yamlpath`'s target key. Set this to a YAML Path that will return the sequence or array that should contain the target key. The test will fail if any element misses the target key.
```
Note, I have used wrong naming for the yaml "items". Hope it is understandable. @evgenyz Feel free to use and abuse the suggestion.

Also, I think it would be good to have examples this is used together with `yaml_path`.

Reply:
Oh, it is hard to explain what this hack does. I'll try to fusion this with my explanations on chat.

Reply:
I'm not sure that the parameter name is clear too.
Suggestion: `key_exists_for`

Reply:
Okay, I hope that I've improved it to some degree

Reply:
This looks great to me.